firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -352,7 +360,7 @@ public TokenResult generateAuthToken(
       @NonNull String fid,
       @NonNull String projectID,
       @NonNull String refreshToken)
-      throws IOException {
+      throws FirebaseInstallationsException, IOException {
```

Comment:
Same question as before.
Btw, just as a comment, no action item for this PR: quite a bit of code duplication going on here.

Reply:
I will address the code duplication in another PR. Updated the exception handling.  PTAL. Thanks